.github/workflows/release.yml

```
@@ -16,15 +16,15 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        go: ['1.16', '1.17', '1.18', '1.19', '1.20']
+        go: ['1.18', '1.19', '1.20']
     name: Go ${{ matrix.go }} test
     steps:
       - uses: actions/checkout@v3
       - name: Setup go
         uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go }}
-      - run: go test -race -v -coverprofile=profile.cov -coverpkg=github.com/zitadel/oidc/... ./pkg/...
+      - run: go test -race -v -coverprofile=profile.cov -coverpkg=./pkg/...,./example/... ./pkg/...
```

perhaps we should skip `./example` here from the coverage report as well. Lots of missed lines and not really relevant.

```suggestion
      - run: go test -race -v -coverprofile=profile.cov -coverpkg=./pkg/... ./pkg/...
```

---

yes i'd go with pkg and remove the example